docs/reference/query-rules/apis/put-query-ruleset.asciidoc

```diff
@@ -0,0 +1,113 @@
+[role="xpack"]
+[[put-query-ruleset]]
+=== Put query ruleset
+
+preview::[]
+
+++++
+<titleabbrev>Put query ruleset</titleabbrev>
+++++
+
+Creates or updates a query ruleset.
+
+[[put-query-ruleset-request]]
+==== {api-request-title}
+
+`PUT _query_rules/<ruleset_id>`
+
+[[put-query-ruleset-prereqs]]
+==== {api-prereq-title}
+
+Requires the `manage_cluster` privilege.
+
+[[put-query-ruleset-path-params]]
+==== {api-path-parms-title}
+
+`<body>`::
+(Required, object)
+Contains parameters for a query ruleset:
+
+====
+`rules`::
+(Required, array of objects)
+The specific rules included in this query ruleset.
+
+Each rule must have the following information:
+
+- `rule_id` (Required, string)
+  A unique identifier for this rule.
+- `type` (Required, string)
+  The type of rule. At this time only `pinned` query rule types are allowed.
+- `criteria` (Required, array of objects)
```

Comment:
I think `actions` is missing from this list?

Reply:
Good catch, thank you!